doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -256,7 +256,7 @@ case class Site(
       blogposts.foreach { file =>
         val BlogPost.extract(year, month, day, name, ext) = file.getName
         val sourceFile = toSourceFile(file)
-        val date = s"$year-$month-$day 00:00:00"
+        val date = s"$month-$day-$year"
```

Comment:
This looks like an ISO date format with the `-` which is supposed to be in little-endian (and ordered). I would prefer to follow the standard.